torch_xla/core/xla_model.py

```diff
@@ -13,6 +13,7 @@
 import torch_xla.utils.utils as xu
 import torch_xla.utils.keyd_queue as kq
 
+MARK_STEP_TOKEN = 'torch_xla.core.xla_model::mark_step'
```

Comment:
Remove this. I know we have two places where the reference is defined, but we don't want to have xla_dist depend on xla_model.